build/sync-translated-content.js

```diff
@@ -73,7 +73,7 @@ function resolve(slug) {
   const url = buildURL("en-us", slug);
   const resolved = Redirect.resolve(url);
   if (url !== resolved) {
-    const doc = Document.read(resolved);
+    const doc = Document.read(Document.urlToFolderPath(resolved));
```

Comment:
Sadly this was broken since March. I missed this in the initial markdown PR (https://github.com/mdn/yari/pull/3138)